src/core/resources/transactions/consolidatedTransactions.ts

```diff
@@ -119,9 +119,15 @@ export async function consolidatedTransactionsQueryFunction({
     };
   } catch (e) {
     // we don't bother with fetching cache and returning stale data here because we probably have previous page data already
-    logger.error(new RainbowError('consolidatedTransactionsQueryFunction: '), {
-      message: e,
-    });
+    if (!(e instanceof Error && e.name === 'AbortError'))
```

Comment:
Wonder if there is a nicer way to apply this globally to react-query or our error boundary to avoid needing piecemeal catching and reporting.